proposals/materialx-versioning/README.md

```diff
@@ -0,0 +1,130 @@
+MaterialX document using the `UsdShade` primitives that have been transported through Hydra (more information 
+available [here](https://openusd.org/release/api/_page__material_x__in__hydra__u_s_d.html)). The reconstructed MaterialX 
+document is then passed to the MaterialX Shader Generation library to generate shader code in the desired destination
+shading languages.
+
+## Problem
+For the entire lifetime of MaterialX support within OpenUSD, there has never been a MaterialX library version change. The 
+MaterialX library version has always been v1.38. The MaterialX project would like to be able to update the MaterialX 
+library version without invalidating MaterialX data currently stored in OpenUSD files. Without access to the MaterialX 
+library version the upgrade code path inside the MaterialX project cannot be used.
+
+Without some sort of remedy, either MaterialX will be constrained to stay at version 1.38, or OpenUSD will become a 
+container for ambiguous MaterialX data.  
+
+## Proposal to Version MaterialX data in OpenUSD
+### Overview
+We propose adding an applied API schema `MaterialXInfoAPI` (in the spirit of the 
+["Revise use of Layer Metadata in USD"](https://github.com/PixarAnimationStudios/OpenUSD-proposals/blob/main/proposals/revise_use_of_layer_metadata/README.md)) that can record the MaterialX library version. We keep the name of the 
+applied API schema a little more generalized to leave the door open for additional MaterialX document information to be
+added to the schema at a later date. We deliberately decide to only propose adding the MaterialX library version to the 
+schema to keep the proposal as uncontroversial as possible to add rapid adoption.
+
+The `MaterialXInfoAPI` will be applied to the UsdShade `Material` primitive. This allows for referencing the material 
+directly from a `.mtlx` file and still retaining the MaterialX library version when the scene is composed. i.e.
+```
+def "Material" MyMaterial (
+    references=@myMaterials.mtlx@</MaterialX/Materials/MyMaterial>
+)
+{}
+```
+
+Any UsdShade `Material` primitives that do not have the applied API would be assumed to be version 1.38, thus providing a 
+path forward to support existing OpenUSD files that are not using this applied API schema.
+
+With the MaterialX library version recorded in the OpenUSD stage, `HdMtlx` would now have access to an appropriate 
+version number to use when re-constructing the MaterialX document. This will then allow the existing MaterialX upgrade 
+functionality to be used.
+
+### UsdMtlx Implementation changes
+
+The only change required in `UsdMtlx` would be to apply the applied API schema to the `Material` primitive, and set the
+MaterialX library version.  This version number would come directly from the `.mtlx` file that was being transcoded to 
+OpenUSD.
+
+### HdMtlx Implementation changes.
+
+The `HdMtlx` library will also need to be changed to read the MaterialX library version from the `Material` primitive, 
```

Comment:
Slight clarification: it will be the usdImaging scene adapter that processes Materials that will need to be updated to pass on the version info, which HdMtlx can then consume from the Hydra dataSource.